src/components/Card.js

```diff
@@ -79,40 +85,78 @@ export const Card = () => {
               value={network.ssid}
               onChange={(e) => setNetwork({ ...network, ssid: e.target.value })}
             />
-            <label className={network.hidePassword ? 'hide-password' : ''}>
+            <label className={network.hidePassword ? 'no-print' : ''}>
               Password
             </label>
             <textarea
               id="password"
               type="text"
+              className={network.hidePassword ? 'no-print' : ''}
               style={{
                 height:
                   portrait && network.password.length > 40 ? '5em' : 'auto',
               }}
-              className={network.hidePassword ? 'hide-password' : ''}
               maxLength="63"
               placeholder="Password"
               autoComplete="off"
               autoCorrect="off"
               autoCapitalize="none"
               spellCheck="false"
               value={network.password}
-              onChange={(e) =>
-                setNetwork({ ...network, password: e.target.value })
-              }
+              onChange={(e) => {
+                setNetwork({ ...network, password: e.target.value });
+              }}
             />
 
-            <input
-              type="checkbox"
-              id="hide-password-checkbox"
-              className="hide-password"
-              onChange={() =>
-                setNetwork({ ...network, hidePassword: !network.hidePassword })
-              }
-            />
-            <label for="hide-password-checkbox" className="hide-password">
-              Hide password field before printing
-            </label>
+            <div className="no-print">
+              <input
+                type="checkbox"
+                id="hide-password-checkbox"
+                onChange={() =>
+                  setNetwork({
+                    ...network,
+                    hidePassword: !network.hidePassword,
+                  })
+                }
+              />
+              <label for="hide-password-checkbox">
+                Hide password field before printing
+              </label>
+            </div>
+
+            <div className="no-print">
+              <label>Encryption:</label>
+              <input
+                type="radio"
+                name="encrypt-select"
+                id="encrypt-none"
+                value="nopass"
+                onChange={(e) => {
+                  setNetwork({ ...network, encryptionMode: e.target.value });
+                }}
+              />
+              <label for="encrypt-none">None</label>
+              <input
```

Comment:
Add a `defaultChecked` attribute for this option to default to selected.

Reply:
Ah, I was trying to use `checked` as the attribute and it was causing issues, so I did not include it. `defaultChecked` works perfectly!